internal/pkg/term/selector/local_selector.go

```diff
@@ -128,23 +129,84 @@ func (s *staticSelector) askCron(scheduleValidator prompt.ValidatorFunc) (string
 // localFileSelector selects from a local file system where a workspace does not necessarily exist.
 type localFileSelector struct {
 	prompt        Prompter
+	ws            *workspace.Workspace
 	fs            *afero.Afero
 	workingDirAbs string
 }
 
 // NewLocalFileSelector constructs a localFileSelector.
-func NewLocalFileSelector(prompt Prompter, fs afero.Fs) (*localFileSelector, error) {
+func NewLocalFileSelector(prompt Prompter, fs afero.Fs, ws *workspace.Workspace) (*localFileSelector, error) {
 	workingDirAbs, err := os.Getwd()
 	if err != nil {
 		return nil, fmt.Errorf("get working directory: %w", err)
 	}
 	return &localFileSelector{
 		prompt:        prompt,
+		ws:            ws,
 		fs:            &afero.Afero{Fs: fs},
 		workingDirAbs: workingDirAbs,
 	}, nil
 }
 
+// StaticSources asks the user to select from a list of directories and files in the current directory and two levels down.
+func (s *localFileSelector) StaticSources(selPrompt, selHelp, anotherPathPrompt, anotherPathHelp string, pathValidator prompt.ValidatorFunc) ([]string, error) {
+	dirsAndFiles, err := s.listDirsAndFiles()
+	if err != nil {
+		return nil, err
+	}
+	if len(dirsAndFiles) == 0 {
+		log.Warningln("No directories or files were found in your workspace. Enter a relative path with the 'custom path' option if you'd like to use a hidden file.")
+	}
+	dirsAndFiles = append(dirsAndFiles, []string{staticSourceUseCustomPrompt}...)
```

Comment:
```suggestion
	dirsAndFiles = append(dirsAndFiles, staticSourceUseCustomPrompt)
```

Reply:
mm, that's just adding the one option to the choices.

Reply:
yeah that's fine - you just don't need the `[]string` wrapper 😊

Reply:
Oh, ha-- I read the suggestion backwards. Thought you were adding it! 🤦🏼